src/layouts/KlientoPage.astro

```diff
@@ -0,0 +1,70 @@
+---
+import { z } from "astro:content";
+import { SecondaryNav } from "../components/TopNav/SecondaryNav";
+import Page from "./Page.astro";
+import klientoLogo from "../assets/images/kliento-logo.png";
+import { primaryNavLinks } from "../components/TopNav/config";
+
+interface Frontmatter {
+  title: string;
+  description: string;
+  service?: string;
+  ogImage?: string;
+  ogType?: string;
+}
+
+interface Props {
+  frontmatter?: Frontmatter;
+  title?: string;
+  description?: string;
+  service?: string;
+  ogImage?: string;
+  ogType?: string;
```

Comment:
Are these actually used? I reckon everything will work if you deleted these attributes.